lib/v2/sehuatang/user.js

```diff
@@ -0,0 +1,104 @@
+// 导入必要的模组
+const got = require('@/utils/got'); // 自订的 got
+const cheerio = require('cheerio'); // 可以使用类似 jQuery 的 API HTML 解析器
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const config = require('@/config').value;
+
+const baseUrl = 'https://sehuatang.org/';
+
+module.exports = async (ctx) => {
+    // 从Url参数中获取uid
+    const uid = ctx.params.uid ?? '186424';
```

Comment:
If you set a default value of a variable, please do mention it in the docs.
docs: https://docs.rsshub.app/joinus/new-rss/add-docs.html
https://docs.rsshub.app/joinus/new-rss/before-start.html

Reply:
Ok, I've removed the default value, it's an unnecessary value.